stripe/test/test_util.py

```diff
@@ -0,0 +1,132 @@
+from __future__ import print_function
+
+try:
+    from unittest import mock
+except ImportError:
+    import mock
+
+from stripe import util
+from stripe.test.helper import StripeUnitTestCase
+
+import __builtin__
+
+PRINT_FUNC_STRING = __builtin__.__name__ + '.print'
+
+
+class DashboardLinkTests(StripeUnitTestCase):
```

Comment:
Yeah, looking at this, the way it is now makes no sense. Originally I had them separate, but after a few refactors it's obvious they should just be in the same class